### ISSUE TITLE
Unable to access full screen feature using the Lucid Theme

### ISSUE THREAD

**Issue by lile53o** [surfbryce/beautiful-lyrics#909]
### How did you install Beautiful Lyrics?

Marketplace

### What is the Bug/Issue?

Unable to click the button for Fullscreen because it registers as clicking the lyric behind it while lucid theme is applied

### What Country are you from?

US

### What Operating System are you using?

Windows

### Spotify/Spicetify Information

Spotify Version: 1.2.47.364.gf06e5cee
Spotify Runtime: cef_128.4.9+g9840ad9+chromium-128.0.6613.120
Spicetify Version: 2.38.4
Spicetify Theme: Lucid / dark
Spicetify Extensions: [shuffle+.js, simple-beautiful-lyrics.js, beautiful-lyrics.mjs, volumePercentageWrapper.js, theme.js]
Spicetify Custom Apps: [marketplace]

### Spotify HTML/CSS and Console Screenshots

[SpotifyHTML.txt](https://github.com/user-attachments/files/17191348/SpotifyHTML.txt)
[SpotifyCSS.txt](https://github.com/user-attachments/files/17191349/SpotifyCSS.txt)
![image](https://github.com/user-attachments/assets/291628c0-c719-4ec5-88e4-13c86b6da6cb)

### Information Checklist

- [X] Spotify/Spicetify Information
- [X] Spotify HTML File
- [X] Spotify CSS File
- [X] Screenshots of Developer Tools Console Tab

**Comment by zydiaa** [comment]
I don't think this is a problem with the Lucid Theme, try reinstalling the extension

**Comment by DataM0del** [comment]
> simple-beautiful-lyrics.js, beautiful-lyrics.mjs
Why do you have both?

**Comment by lile53o** [comment]
I uninstalled simple beautiful lyrics and reinstalled Beautiful lyrics and still have the same issue
